client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -200,12 +248,52 @@ export default React.createClass( {
 			} );
 		}
 		renewLink = this.renewLink( expiringDomains.length );
-		return <Notice
-			isCompact={ this.props.isCompact }
-			status="is-error"
-			showDismiss={ false }
-			key="expiring-domains"
-			text={ text }>{ renewLink }</Notice>;
+
+		return (
+			<Notice
+				isCompact={ this.props.isCompact }
+				status="is-error"
+				showDismiss={ false }
+				key="expiring-domains-can-manage"
+				text={ text }>
+				{ renewLink }
+			</Notice>
+		);
+	},
+
+	expiringDomainsCannotManage() {
+		let text;
+		const expiringDomains = this.getDomains().filter( domain => domain.expirySoon && domain.type === domainTypes.REGISTERED && ! domain.currentUserCanManage );
+
+		if ( expiringDomains.length === 0 ) {
+			return null;
+		}
+		if ( expiringDomains.length === 1 ) {
+			text = this.translate( 'The domain %(domainName)s will expire in %(timeUntil)s. It can be renewed by the user {{strong}}%(owner)s{{/strong}}.', {
```

Comment:
AFAICT, you don't need the `in` part in `will expire in %(timeUntil)s.` - it will be included by `moment`. See the case for expired domains above.